### PR TITLE
Add option to copy PR URL to clipboard

### PR DESCRIPTION
**Pull request by Caleb-T-Owens**
My workflow currently requires me to copy the URL of my PRs to put them into Trello. Having to follow the link to GH to then copy it is quite cumbersome so I've added in a context menu that has an option to copy the link to the clipboard

### CODE REVIEW

File: gitbutler-ui/src/lib/components/ActiveBranchStatus.svelte [hunk 1]
```diff
@@ -11,6 +13,22 @@
...
+
+	function updateContextMenu() {
+		if (popupMenu) popupMenu.$destroy();
+		return new ViewPrContextMenu({
+			target: document.body,
+			props: { prUrl: prUrl || '' }
+		});
+	}
+
+	$: popupMenu = updateContextMenu();
```
Comment (mtsgrd) [inline gitbutler-ui/src/lib/components/ActiveBranchStatus.svelte]: I haven't tested this out, but it looks like we ought to pass `prUrl` as a parameter to this function to make it react to new values?

Comment (Caleb-T-Owens) [reply gitbutler-ui/src/lib/components/ActiveBranchStatus.svelte]: Thanks for pointing that out - I think you're probably right

**Comment by mtsgrd** [comment]
Perfect, thank you!

As a side note, I'm starting to wonder if it'd be better to have one instance of each type of context menu living closer to the root in the dom.. that way we wouldn't need to render them dynamically.

**Comment by Caleb-T-Owens** [comment]
@mtsgrd I was thinking perhaps something one step further would be better. I was wondering if it would make sense to have just one context menu at the top that you could pass an object which contained all the different options and handlers and it would dynamically render it out

So you'd just make one call like when you want to have a contextMenu
```ts
<MyThing
on:contextmenu={(e) => {
  openContextMenu({
    "Copy to Clipboard": () => { copyToClipboard(prUrl) }
  })
}}>
</MyThing>
```

**Pull request merged**